bundles/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/NetatmoGenericBindingProvider.java

```diff
@@ -203,10 +393,18 @@ public void processBindingConfiguration(final String context, final Item item, f
         NetatmoMeasureType measureType;
         NetatmoScale netatmoScale;
 
+        // Netatmo welcome
+        String homeId;
+        String personId;
+        String cameraId;
+        NetatmoWelcomeAttributes attribute;
+
         @Override
         public String toString() {
             return "NetatmoBindingConfig [userid=" + this.userid + ", deviceId=" + this.deviceId + ", moduleId="
-                    + this.moduleId + ", measure=" + this.measureType.getMeasure() + "]";
+                    + this.moduleId + ", measure=" + (this.measureType != null ? this.measureType.getMeasure() : null)
+                    + ", homeId=" + this.homeId + ", personId=" + this.personId + ", cameraId=" + this.cameraId
+                    + ", attribute=" + this.attribute + "]";
```

Comment:
Check if its a weather or a camera item and just write the items which belongs to the type